modules/manually-create-identity-access-management.adoc

```diff
@@ -0,0 +1,178 @@
+This removal prevents your admin credential from ever being stored in the
+cluster:
++
+----
+$ rm mycluster/openshift/99_cloud-creds-secret.yaml
+----
+
+. Obtain the {product-title} release image your `openshift-install` binary is built
+to use:
++
+----
+$ bin/openshift-install version
+release image quay.io/openshift-release-dev/ocp-release:4.z.z-x86_64
+----
+
+. Locate all `CredentialsRequests` in this release image that target the cloud you
+are deploying on:
++
+----
+$ oc adm release extract quay.io/openshift-release-dev/ocp-release:4.z.z-x86_64 --to ./release-image
+----
+
+. Locate the `CredentialsRequests` in the extracted file:
++
+----
+$ grep -l "apiVersion: cloudcredential.openshift.io" * | xargs cat
+----
++
+[NOTE]
+====
+In a future {product-title} release, there will be a new `oc adm release`
+command to scan for the `CredentialsRequests` and display them.
+====
++
+This displays the details for each request. Remember to ignore any
+`CredentialsRequests` where the `spec.providerSpec.kind` does not match the cloud
+provider you are installing to.
++
+ifdef::aws[]
+.Sample CredentialsRequest
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: cloud-credential-operator-iam-ro
+  namespace: openshift-cloud-credential-operator
+spec:
+  secretRef:
+    name: cloud-credential-operator-iam-ro-creds
+    namespace: openshift-cloud-credential-operator
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AWSProviderSpec
+    statementEntries:
+    - effect: Allow
+      action:
+      - iam:GetUser
+      - iam:GetUserPolicy
+      - iam:ListAccessKeys
+      resource: "*"
+----
+endif::aws[]
+ifdef::azure[]
+.Sample CredentialsRequest
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: openshift-image-registry-azure
+  namespace: openshift-cloud-credential-operator
+spec:
+  secretRef:
+    name: installer-cloud-credentials
+    namespace: openshift-image-registry
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AzureProviderSpec
+    roleBindings:
+    - role: Contributor
+----
+endif::azure[]
+ifdef::google-cloud-platform[]
+.Sample CredentialsRequest
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: openshift-image-registry-gcs
+  namespace: openshift-cloud-credential-operator
+spec:
+  secretRef:
+    name: installer-cloud-credentials
+    namespace: openshift-image-registry
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: GCPProviderSpec
+    predefinedRoles:
+    - roles/storage.admin
+    - roles/iam.serviceAccountUser
+    skipServiceCheck: true
+----
+endif::google-cloud-platform[]
+
+. Create secret YAML files in the `openshift-install` manifests directory that you
```

Comment:
Maybe "Create YAML files for secrets [...]", so that it doesn't sound like the files themselves are secret. :detective: